helix-lsp/Cargo.toml

```diff
@@ -23,5 +23,5 @@ lsp-types = { version = "0.89", features = ["proposed"] }
 serde = { version = "1.0", features = ["derive"] }
 serde_json = "1.0"
 thiserror = "1.0"
-tokio = { version = "1.9", features = ["rt", "rt-multi-thread", "io-util", "io-std", "time", "process", "macros", "fs", "parking_lot"] }
-tokio-stream = "0.1.7"
+tokio = { version = "1.9", features = ["rt", "rt-multi-thread", "io-util", "io-std", "time", "process", "macros", "fs", "parking_lot", "sync"] }
+tokio-stream = "0.1"
```

Comment:
Not sure whether these corrections belong here really

Reply:
Possibly we can even find those variables in code using LSP and show diagnostics near them for values obtained via debug.

Reply:
I added them since I realized the dap one was invalid (missing sync), and it was copied from the lsp one.